src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -10,12 +12,14 @@
 public class DriverManager {
 
 	private Job2dDriver currentDriver = new LoggerDriver();
+	private final Publisher publisher = new Publisher();
```

Comment:
Sorry, but adding observer capabilities to DriverManager was not your task. We are still waiting for the solution from a different group. So either, simplify your solution or we will wait for this feature to be delivered.

Reply:
I dodn't notice that there is separate story for that. Also, there was a tip in description to use observer pattern. Removed and waiting for implementation.